l1-contracts/deploy-scripts/DeployL2Contracts.sol

```diff
@@ -41,6 +43,27 @@ contract DeployL2Script is Script {
         deployFactoryDeps();
         deploySharedBridge();
         deploySharedBridgeProxy();
+        deployForceDeployer();
+
+        saveOutput();
+    }
+
+    function runDeploySharedBridge() public {
```

Comment:
I think you can probably get rid of this ( runDeploySharedBridge). 

However we are not actually re-adding it, so maybe it can stay. 
https://github.com/matter-labs/era-contracts/pull/463/files#diff-72cab2f3f48ef3d515f8456f9654e055b7dd05e1caaa21ee373e72dbb2dd55da